tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/contact/EmailAddressContact.scala

```diff
@@ -27,9 +27,14 @@ case class ContactFields(address: MailAddress, firstname: String = "", surname:
     address.asString().contains(part) || firstname.contains(part) || surname.contains(part)
 }
 
-case class AccountEmailContact(accountId: String, id: UUID, address: MailAddress) {
-  def this(accountId: AccountId, emailAddressContact: EmailAddressContact) =
-    this(accountId.getIdentifier, emailAddressContact.id, emailAddressContact.fields.address)
+case class UserContactDocument(accountId: String, contactId: UUID, email: MailAddress, firstname: String, surname: String) {
```

Comment:
Should we have a trailt ContactDocument class, that has common properties?
Then we has two implement/extend: UserContact and DomainContact